src/middlewares/verifyToken.js

```diff
@@ -0,0 +1,20 @@
+const admin = require('../config/firebase'); // Ensure path is correct
+
+const verifyToken = async (req, res, next) => {
+    const idToken = req.headers.authorization && req.headers.authorization.split('Bearer ')[1];
+    
+    if (!idToken) {
+        return res.status(401).json({ error: 'No token provided' });
```

Comment:
This should be a `400` (which indicates the request object was not valid that the client sent) to differentiate it from when a token is not valid.